Corale.Colore/Razer/Keyboard/Effects/Custom.cs

```diff
@@ -86,7 +86,11 @@ public Custom(Custom other)
             for (var index = 0; index < Constants.MaxKeys; index++)
             {
                 _colors[index] = other[index];
-                _keys[index] = other[(Key)index];
+                var otherKeyColor = other[(Key)index];
+                if (otherKeyColor.Value > 0)
+                {
+                    _keys[index] = Constants.KeyFlag | otherKeyColor;
```

Comment:
This will force the key enabled, even if previously disabled.

E.g.: `0x00FFFFFF` would be copied as `0x01FFFFFF`.

Probably better to use raw value here as well.